README.md

```diff
@@ -0,0 +1,62 @@
+# Betterbird (Flathub)
```

Comment:
Just to clarify: no regular user will ever read this file. If you consider it important documentation, it should be on your website.